pkg/multicluster/kubeconfig/predicate.go

```diff
@@ -0,0 +1,39 @@
+package kubeconfig
+
+import (
+	"reflect"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+func Predicate() predicate.Predicate {
```

Comment:
how about using a `var Predicate = ` here instead of a `func`?

Reply:
as a convention for storing global types (e.g. it allows modifyting the `Predicate` for testing)